### DISCUSSION TITLE
Does this works on Android with a closed PWA?

### DISCUSSION THREAD

**Discussion by PlopTheReal** [ali-master/pwa-badge#3] [Q&A] [answered]
I've been testing navigator.setAppBadge on a service worker running against a closed PWA on Android but it doesn't seems to work (no badges are shown while the call to the API is successful).
Does this library works within such scenario?

**Comment by ali-master** [comment, marked as answer]
Now, it is just supported by the macOS and Windows and will be supported in Android and iOS soon.

**Reply by PlopTheReal** [reply]
Thanks, any guess on a date?

**Reply by ali-master** [reply]
You can see all information that you need here: https://caniuse.com/?search=badge
